### ISSUE TITLE
When I try to use manifest url, it says "The 'path' argument must be of type string. Received type undefined"

### ISSUE THREAD

**Issue by glaucolessa** [Reaver01/ryuutama#63]
I had some difficulty finding the manifest URL. I tried https://raw.githubusercontent.com/reaver01/ryuutama/master/system.json

If that's right, it says the error on the title. Maybe it's another .json file?

It would help a lot to see the manifest URL in the read me file. And if it's something wrong about it, I hope it may be fixed soon.

I just discovered Ryuutama and it seems an amazing game. Thanks a lot for the module! I hope I can use it soon!

Regards!

**Comment by Reaver01** [comment]
https://foundryvtt.com/packages/ryuutama/

The system should be able to be installed just by searching for it. That is why I never listed the manifest in the GitHub.

**Comment by glaucolessa** [comment]
> https://foundryvtt.com/packages/ryuutama/
> 
> The system should be able to be installed just by searching for it. That is why I never listed the manifest in the GitHub.

Hello!

I noticed I was in the modules part, not systems. It was only my fault hahaha

Before coming here to tell, I just pasted the URL and worked, but I just saw your answer and realized I could have done it even easier.

I searched for ryuutama in the modules, found nothing and googled it to get here. In the end, if I had looked in the right place, everything would be ok hahah


Sorry about that, and thanks again. Checking out the sheet and monsters, it's very well done :) good work!
